src/Dfe.PlanTech.Infrastructure.SignIn/ConnectEvents/OnUserInformationReceivedEvent.cs

```diff
@@ -23,11 +21,6 @@ public static async Task OnUserInformationReceived(UserInformationReceivedContex
         await RecordUserSign(context);
 
         var config = context.HttpContext.RequestServices.GetRequiredService<IDfeSignInConfiguration>();
```

Comment:
this config var is no longer used, which makes the function just `await RecordUserSign` which is used in just one place
I reckon we delete this function and just directly put `RecordUserSign` (& rename Signin) into the `OpenIdConnectEvents`